app/scripts/directives/multiSelectMenu.js

```diff
@@ -58,17 +58,17 @@ angular.module('gsUiInfraApp')
                 }
 
                 /**
-                 * Bind data to ng-model
+                 * call onchange function when model change
```

Comment:
is this an optimization? or a fix? the title suggests a fix. what was wrong before?

how does this behave if we change the onchange value?